lib/hutch/acknowledgements/nack_on_all_failures.rb

```diff
@@ -0,0 +1,18 @@
+require 'hutch/logging'
+
+module Hutch
+  module Acknowledgements
+    class NackOnAllFailures
+      include Logging
+
+      def handle(delivery_info, properties, broker, ex)
```

Comment:
Scratch the above. It would lead to inconsistency with error handlers, which use `handle` and it is way too late to change that.